edb/server/pgcluster.py

```diff
@@ -966,6 +967,13 @@ async def _detect_capabilities(
             )
         except pgcon.BackendPrivilegeError:
             can_make_superusers = False
+        except pgcon.BackendError as e:
+            if e.code_is(errors.ERROR_INTERNAL_ERROR):
+                # DigitalOcean raises a custom error:
+                # XX000: Role ... not in permitted superuser list
```

Comment:
I think we should also match the error message, since this is very specific behavior and we probably don't want to mask other errors like this.